tools/pythonpkg/src/include/duckdb_python/pyrelation.hpp

```diff
@@ -93,7 +93,7 @@ struct DuckDBPyRelation {
 	static unique_ptr<DuckDBPyRelation> FilterDf(const py::object &df, const string &expr,
 	                                             DuckDBPyConnection *conn = DuckDBPyConnection::DefaultConnection());
 
-	unique_ptr<DuckDBPyRelation> Limit(int64_t n);
+	unique_ptr<DuckDBPyRelation> Limit(int64_t n, int64_t offset = 0);
```

Comment:
This should be an unsigned, no?

Reply:
The main C++ code
https://github.com/duckdb/duckdb/blob/ac159c1673f87f0cd489a12eb024c446e7ace285/src/include/duckdb/main/relation.hpp#L86
uses an `int64_t`, which is why I chose the same here. Negative values seem to be ignored in https://github.com/duckdb/duckdb/blob/master/src/main/relation/limit_relation.cpp